components/icons/Modelina.js

```diff
@@ -0,0 +1,7 @@
+export default function IconGenerator ({ className }) {
+  return (
+    <svg className={className} strokeWidth=".3" fill="currentColor" viewBox="0 0 20 20">
```

Comment:
The icon is not well aligned so I propose you fix it like this:

```suggestion
    <svg className={className} strokeWidth=".3" fill="currentColor" viewBox="1.5 1 20 20">
```

Reply:
Thanks!